Sources/PointFree/StripeWebhooks.swift

```diff
@@ -21,9 +21,12 @@ let stripeWebhookMiddleware: Middleware<
   Data
   >
   = validateStripeSignature
+    <<< filter(
+      { $0.data.object.either({ $0.number != nil }, const(true)) },
```

Comment:
is it possible to write a middleware test for this?

Reply:
Done.